### PR TITLE
TI dashboard UX finishing touches

### LINKED ISSUE #6215: Finalize TI Dashboard improvements
**Describe the solution:** 
Do a final verification of the new UI for the TI Dashboard to make sure that it matches the mockups.   These UI changes are so that CBO members are able to more easily navigate through the information.  

Important Note:  This issue should be worked on **last** as a final verification that all the components fit together correctly.  Almost all of the layout will be implemented in other issues before this one.  

**Figma mocks:** [mocks](https://www.figma.com/file/yfhnUmE2FsonLtAqfeEfoo/TI-Edit?type=design&node-id=1%3A19&mode=design&t=TWL8Jxmrv2Q5wkVP-1)

**Screenshots:**

Image

**Done when:** 
* UX has approved all the TI Dashboard improvements
* All buttons use the USWDS styles
* Page layout and components look like mocks above
* CBO member sees and can use "Search" section
* CBO member sees "Displaying all clients" header
* Clicking `Add new client` button opens the page to add a new client 
* All clients are displayed in a list in order of creation (same behavior we have now) 
* Up to 10 client cards displayed at a time
* Pagination at the bottom allows CBO member to click through pages of clients, either by clicking on the page number itself or by clicking `Next`

**Additional information:** 
Beyond the actual layout of the page, there are a lot of related tickets for individual components needed for this page to line up with mocks: 
* Search + datepicker details in #6214 
* Individual card component in #6212 
* Pagination details in #6211

### PR DESCRIPTION
**Pull request by melanie-exygy**
### Description

Made the following TI dashboard styling changes to match the mocks:
- Background color is now white rather than light grey
- TI page headers are enclosed in bands of light grey
- Page container extends to screen width
- Search button is outline style rather than default blue
- "Search by date of birth" is now sentence-case

image

image

### Instructions for manual testing

1. Log in as a Trusted Intermediary.  
2. Note the visual differences, especially the background colors.

### Issue(s) this completes

Fixes #6215

### CODE REVIEW

File: browser-test/image_snapshots/trusted_intermediary_test/add-client-invalid-dob.png
[the remark this answers is not in the document]
Comment (azhangggggg) [reply browser-test/image_snapshots/trusted_intermediary_test/add-client-invalid-dob.png:1]: Yeah it's showing up with bands of grey on my side as well. Thanks for the note.
Screenshot 2024-04-04 at 3 01 01 PM

File: server/conf/i18n/messages [hunk 1]
```diff
@@ -215,7 +215,7 @@ label.notes=Notes
 # Label above the phone number input field when creating or editing a client.
 label.phoneNum=Phone number
 # The label above the Date of Birth search input for the client search.
-label.searchByDob=Search by Date of Birth
+label.searchByDob=Search by date of birth
```
Comment (azhangggggg) [inline server/conf/i18n/messages:218]: A note for the next translation batch - transifex might attempt changing `label.searchByDob` in translated message files back to its english form, but we need to overwrite that when doing translation batch since it's just a case change.

File: server/app/views/applicant/TrustedIntermediaryDashboardView.java [hunk 2]
```diff
@@ -99,21 +100,32 @@ public Content render(
...
+                    .withClasses(
+                        "flex",
+                        "justify-between",
+                        "items-center",
+                        "py-4",
+                        "px-20",
+                        "ti-header-band",
+                        "mb-10"),
+                div(
+                        h4("Search"),
```
Comment (azhangggggg) [inline server/app/views/applicant/TrustedIntermediaryDashboardView.java:117]: I think there's a localized message for this `Search`, but you can totally do it in a different PR.

**Review by azhangggggg — APPROVED** [review]

File: server/app/views/applicant/TrustedIntermediaryDashboardView.java (hunk 2 above)
Comment (melanie-exygy) [reply server/app/views/applicant/TrustedIntermediaryDashboardView.java:117]: Could callout.  I'm planning on doing a final check that all text is translated as part of this issue:  #7046
